docs/how-to/manage-relationships

```diff
@@ -0,0 +1,78 @@
+`Relationship` is the word used by Cozy-Client, to refer relation / association between documents. For that, we need to define our schema.
+
+Ex: 
+```jsx
+const schema = {
+    todos: {
+       doctype: 'io.cozy.todos',
+       relationships: {
+         tasks: {
+           doctype: 'io.cozy.tasks',
+           type: 'has-many'
+         }
+       }
+     }
+  }
+```
+
+Here, it's say that our `todos` document can be associated to `has-many` `tasks` document.
+
+#How to access todos with their associated tasks ? 
+
+Pretty simple, first let's create a `Query` to fetch our `todos` 
+
+```jsx
+<Query
+  query={client =>
+    client.find('io.cozy.todos')
+  }
+>
+{({ data: todos, fetchStatus }) => {
+    if (fetchStatus === 'loading') {
+      return 'loading'
+    }
+    todos.map(todo => console.log(todo))
+    return null
+  }}
+</Query>
+```
+
+Our `todo` object will have its own properties, but also its tasks. 
+
+See, we have a `tasks` key. `tasks` is the name we gave to our relationship when defining it inside our schema. We can name it as you want but changing the key in your schema. 
```

Comment:
```suggestion
See, we have a `tasks` key. `tasks` is the name we gave to our relationship when defining it inside our schema.
```

I prefer not to give potential confusing information.

Reply:
Agreed but without any explanation `tasks` key is a bit obscur. Nop?

Reply:
The key name describe at the same time where is stored the relationship (in the case of an has-many in `.relationships[relationshipName]`, in case of has-many-in-place in `.[relationshipName]`) in the original document and where it will be stored in the parsed document.

I would prefer to have something simpler, maybe we can hide this behind a details tag ?